Configuration/Eras/python/Era_Run3_ecalClustering_cff.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run3_cff import Run3
+from Configuration.Eras.Modifier_run3_ecalclustering_cff import run3_ecalclustering
+
+Run3_ecalClustering = cms.ModifierChain(Run3,run3_ecalclustering)
```

Comment:
To me `run3_ecalclustering` looks more like a technical modifier to switch on a specific MixingModule behavior than a feature depending on the data taking conditions. "Run 3" looks like a use case for using the behavior, e.g. is there any fundamental reason why the behavior would not need to be enabled for Run 2 or Phase 2?

For technical modifiers the `Configuration.ProcessModifiers` is more proper place. I'd also suggest the name to be something reflecting behavior, e.g. `ecal_caloParticles`.

Reply:
All these changes are needed for ECAL clustering studies that we are doing right now, aiming to Run3. Other PRs will be done soon, which change ECAL PFClusters and will use this same era.

Reply:
Will the other PRs depend on the CaloParticles being produced?

Reply:
They won't depend on CaloParticles, but will change PFClustering and we want to apply that in this customization (Era_Run3_ecalClustering)

Reply:
@bmarzocc I agree with @makortel , this change does not qualify a new different Era, it is a possible technical modification of one or more of the many eras we have. If you intend to apply this from run3 onwards, it will be a modifier to be chained there with all the others already existing

Reply:
@fabiocos my point is that WE DO NOT WANT to affect anything else with our changes, as these are studies for Run3. Therefore we don't want to modify Eras that are already there for the moment. Because of this, we decided to make a completely new Era so that we are sure we are fully independent to any other development. Is there any other way to that instead of making a new era?

Reply:
The technical modifiers in `Configuration.ProcessModifiers` can be enabled independently from the eras in `Configuration.Eras` in python by
```python
from Configuration.Eras.Era_Run3_cff import Run3
from Configuration.ProcessModifiers.run3_ecalClustering
process = cms.Process("FOO", Run3, run3_ecalClustering)
```
or with cmsDriver by
```bash
cmsDriver.py ... --era Run3 --procModifiers run3_ecalClustering ...
```

Reply:
ok, thanks for that! I will do as suggested.